### ISSUE TITLE
Exception!

### ISSUE THREAD

**Issue by owldefender** [soulrelic616/bfc1138.js#16]
![error](https://cloud.githubusercontent.com/assets/16391066/11941074/95c53606-a7f2-11e5-94a3-8e9733900785.png)

I can do ?, I get when I win / I lose battles, or appears that If      :c

**Comment by soulrelic616** [comment]
Is the issue random?  OR does it happen in certain planets only? 

Can you please give us your tech specs? Thanks!

**Comment by owldefender** [comment]
Sometimes it happens when you start or end a fight , it happens on planets like bespin , Kuat, Alderaan, etc , but also occurs in any planet or when I want to save the game. :c

**Comment by soulrelic616** [comment]
Try moving all your other mods to a different location, could be a conflict

**Comment by SirWalkAlot** [comment]
Have you installed any unofficial RAM patches? if not you should download one. That worked for me.

**Comment by owldefender** [comment]
Remove my mods, and is still happening , where I download the patch?  Sirwalk?

**Comment by HelloImMoonMoon** [comment]
i have the same problem, it is happening all the time, i dont have other mods for this game, i also have the steam version, windows 10

**Comment by HelloImMoonMoon** [comment]
hey it seems this is the reason, and there is an unofficial patch to solve this problem, but i cant get it :/ if you can, pls share it with us! the problem is running the game in a 64 bit system 

http://www.techsupportforum.com/forums/f59/solved-star-wars-empire-at-war-forces-of-corruption-crashes-when-launching-679877.html

**Comment by SirWalkAlot** [comment]
First download all patches and fixes on Disney's official site : https://help.disney.com/articles/en_US/FAQ/STAR-WARS-Empire-at-War-Update-Files?section=Star-Wars#fix
If that didn't help try this unofficial patch:http://www.gamefront.com/files/20635580(just unzip the file and run the update)

**Comment by HelloImMoonMoon** [comment]
ok, the official site gave me a notificacion a few times, an error ´´unable to write the installation folder.´´ a few attempts and then it finally installed. the second one, the unofficial patch is not there anymore :/ im going to try if the disney patch solve the problem, and i think the problem is in the mod, cuz id been playing forces of corruption gold from steam the last couple of hours and nothing wrong happened. the same problem as republic at war, maybe is the GC map, a shame, its my favorite map of all those mods.

**Comment by owldefender** [comment]
with mod Republic at war does not happen to me. They did not work the 2 methods

**Comment by HelloImMoonMoon** [comment]
yes! that solve the problem, thanks!! the game crash me just one time yesterday, but i played all day and nothing more happened, before the patch the game was crashing every 5-10 minutes.

**Comment by owldefender** [comment]
that's what you did, helloIm ?. I'm still the problem   :c
